src/generator/backend/generator.js

```diff
@@ -7,6 +7,7 @@ const moment = require('moment');
 const handlebars = require('handlebars');
 const async = require('async');
 const archiver = require('archiver');
+const fold=require('./fold.js');
```

Comment:
also prefer using (__dirname + '/fold.js') as we never know where the generator.js is called from. __dirname ensures dirpath starts from the folder containing this file